pkg/alertmanager/amcfg.go

```diff
@@ -128,6 +128,56 @@ func validateConfigInputs(am *monitoringv1.Alertmanager) error {
 	return nil
 }
 
+func (cg *configGenerator) generateGlobalConfig(
+	ctx context.Context,
+	amConfig *monitoringv1alpha1.AlertmanagerConfig,
+) (*alertmanagerConfig, error) {
+	globalAlertmanagerConfig := &alertmanagerConfig{
+		Receivers: []*receiver{
+			{
+				Name: "null",
+			},
+		},
+		Route: &route{
+			Receiver: "null",
+		},
+	}
```

Comment:
We shouldn't pre-set any fields.
```suggestion
	globalAlertmanagerConfig := &alertmanagerConfig{}
```